ERROR 2023-09-07 06:33:45,253 urls Relative URI reference without a base URI: <img src="logo.png">
ERROR 2023-09-07 06:33:47,165 urls Relative URI reference without a base URI: <img src="logo.png">
ERROR 2023-09-07 06:33:56,250 urls Relative URI reference without a base URI: <img src="logo.png">
ERROR 2023-09-07 06:33:57,894 urls Relative URI reference without a base URI: <img src="logo.png">
ERROR 2023-09-07 06:38:17,834 urls Relative URI reference without a base URI: <img src="logo.png">
ERROR 2023-09-07 06:38:19,824 urls Relative URI reference without a base URI: <img src="logo.png">
ERROR 2023-09-07 06:42:49,578 views 'Email\r\na'
Traceback (most recent call last):
  File "/home/rifat/Projects/django/workstation/django_smtp_dynamic_template/smtp_send_mail/smtp_send_mail/views.py", line 460, in generate_updates
    content_body = content_body.replace('#'+tag, str(data[formated_tag]))
                                                     ~~~~^^^^^^^^^^^^^^
KeyError: 'Email\r\na'
